gems/pending/appliance_console/locales/en.yml

```diff
@@ -27,6 +28,7 @@ en:
     httpdauth: Configure External Authentication (httpd)
     evmstop: Stop EVM Server Processes
     key_gen: Generate Custom Encryption Key
+    scap: Apply SCAP Compliant Appliance Configuration
```

Comment:
@Fryguy  can you give me a little more background on this option?  I do think that we probably want to work the word "Harden" into the title.